automated install/basic-install.sh

```diff
@@ -259,6 +259,29 @@ os_check() {
     fi
 }
 
+# This function waits for dpkg to unlock, which signals that the previous apt-get command has finished.
+test_dpkg_lock() {
+    i=0
+    printf "  %b Waiting for package manager to finish\\n" "${INFO}"
```

Comment:
```suggestion
    printf "  %b Waiting for package manager to finish (up to 30 seconds)\\n" "${INFO}"
```

Or something like this?

Reply:
Happy to add this after we agreed on a time

Reply:
I think 30 seconds is probably fine - I just wonder if there are any legitimate processes we could end up killing/cutting short if we exit after 30 seconds - but I may be thinking about it the wrong way around

Reply:
We do not kill any other process. If 30 seconds have passed, we exit our installer. The other process is allowed to keep the lock.

Reply:
Right, in that case I totally got the wrong end of the stick here - and I think 30 seconds is fine!